lib/bundles/inspec-compliance/cli.rb

```diff
@@ -57,7 +57,14 @@ def login(server) # rubocop:disable Metrics/AbcSize
       puts '', msg
     end
 
-    desc "login_automate SERVER --user='USER' --ent='ENT' --dctoken or --usertoken='TOKEN'", 'Log in to an Automate SERVER'
+    desc "login_automate SERVER --insecure --user='USER' --ent='ENT' --usertoken='TOKEN'", 'Log in to an Automate SERVER'
```

Comment:
I would love to see this be `inspec compliance login --automate` in the future than require a completely separate command. It feels weird to me from a CLI UX perspective. Not a blocker, obviously :)